clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -119,6 +145,17 @@ public boolean ready(Node node, long now) {
     }
 
     /**
+     * Closes the connection to a particular node (if there is one).
+     *
+     * @param nodeId The id of the node
+     */
+    @Override
+    public void close(String nodeId) {
```

Comment:
I named this `close` instead of `disconnect` to match the relevant method in `Selector`, but I'd be interested in feedback regarding the naming. Also, it's not clear to me when `Selector.disconnect` is actually useful.